.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```diff
@@ -118,22 +124,33 @@ stages:
                   envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY}' < test/integration/manifests/cilium/v${DIR}/cilium-operator/templates/deployment.yaml | kubectl apply -f -
                   kubectl get po -owide -A
 
-                  if [ -z "$TEST_AZURE_IPAM_VERSION" ]
-                  then
-                    echo "TEST_AZURE_IPAM_VERSION is not set, using default value"
-                    IPAM=$(make azure-ipam-version)
-                  else
-                    IPAM=$(TEST_AZURE_IPAM_VERSION)
-                    echo "TEST_AZURE_IPAM_VERSION is set to $IPAM"
-                  fi
-                  if [ -z "$TEST_CNS_VERSION" ]
+                  if [ ${{parameters.upgradeScenario}} = "true" ]
                   then
-                    echo "TEST_CNS_VERSION is not set, using default value"
-                    CNS=$(make cns-version)
+                    echo "Upgrade scenario is true, using upgrade azure ipam and cns version from pipeline variables"
+                    IPAM=$(UPGRADE_AZURE_IPAM_VERSION)

```

Comment:
Even if `upgradeAzureIpam` is false, will we use the `UPGRADE_AZURE_IPAM_VERSION` ?